src/App.tsx

```diff
@@ -0,0 +1,54 @@
+import './App.css';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import { Box, Button } from '@mui/material';
+import { useState } from 'react';
+
+import { Main } from './components';
+import { getDataConfig } from './utils/getDataConfig';
+import { Header } from './pages/Header';
+import { SideBarLeft } from './pages/SideBarLeft';
+
+function App() {
+  const [isDrawerShow, setIsDrawerShow] = useState(false);
+  const data = getDataConfig();
+  const handleDrawerClose = (): void => {
+    setIsDrawerShow(false);
+  };
+  const handleDrawerOpen = (): void => {
+    setIsDrawerShow(true);
+  };
+
+  return (
+    <BrowserRouter>
+      <Box>
+        <Header open={isDrawerShow} onClick={handleDrawerOpen} />
+        <SideBarLeft
+          isDrawerShow={isDrawerShow}
+          handleDrawer={handleDrawerClose}
+          listItems={data}
+        ></SideBarLeft>
+        <Main open={isDrawerShow}>
+          <Routes>
+            <Route
+              path="/"
+              element={<Button variant="contained">Home</Button>}
+            />
+            {data.map(({ rootApp, sidebarItem }) => (
+              <Route
+                key={'route-' + rootApp}
```

Comment:
https://track.astral.ru/soft/wiki/pages/viewpage.action?pageId=3557425260#id-%D0%9F%D1%80%D0%B8%D0%B2%D0%B5%D0%B4%D0%B5%D0%BD%D0%B8%D0%B5%D1%82%D0%B8%D0%BF%D0%BE%D0%B2-%D0%94%D0%BB%D1%8F%D1%81%D0%BA%D0%BB%D0%B5%D0%B8%D0%B2%D0%B0%D0%BD%D0%B8%D1%8F%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BD%D0%B5%D0%BE%D0%B1%D1%85%D0%BE%D0%B4%D0%B8%D0%BC%D0%BE%D0%B8%D1%81%D0%BF%D0%BE%D0%BB%D1%8C%D0%B7%D0%BE%D0%B2%D0%B0%D1%82%D1%8C%D1%81%D1%82%D1%80%D0%BE%D0%BA%D0%BE%D0%B2%D1%8B%D0%B9%D1%88%D0%B0%D0%B1%D0%BB%D0%BE%D0%BD